build_prod.py

```diff
@@ -210,9 +254,25 @@ def add_meta_layers(cfg):
             yocto_add_bblayer(cfg, bblayer)
 
 
-def build_init(uri, branch, xml_base_name):
+def build_init(uri, branch, xml_base_name, proposed):
+    if not branch:
+        branch = 'master'
+
     repo_init(uri, branch, xml_base_name)
     repo_sync()
+    
+    c_dir = os.getcwd()
```

Comment:
Could you please add a comment explaining what the magic here is

Reply:
> Could you please add a comment explaining what the magic here is

added